verify.go

```diff
@@ -0,0 +1,215 @@
+/*
+ * knoxite
+ *     Copyright (c) 2020, Fabian Siegel <fabians1999@gmail.com>
+ *
+ *   For license see LICENSE
+ */
+
+package knoxite
+
+import (
+	"math"
+	"math/rand"
+)
+
+func VerifyRepo(repository Repository, Percentage int) (prog chan Progress, err error) {
```

Comment:
Being a regular parameter here, we should use the lowercase version `percentage`.